docs/logging/README.md

```diff
@@ -0,0 +1,130 @@
+# Logging stack
+
+### Why do we need logging stack?
+We need logging stack because currently Kubernetes use the underlying container runtime logging which does not persist log for not existing containers. This makes difficult to investigate issues ocurred in long ago destroyed containers.
+Our logging stack is created to keep logs for 14 days. Also there are pretty UI which can be used for easier log filtering.
+
+
+### Architecture:
+* Fluent-bit Deamonset which works like a log collector. Also we use custom Golang plugin which spreads log messages to their Loki instances
+* One Loki Statefulset in the `garden` namespace which contains logs for the seed cluster and one per shoot namespace which contains logs for shoot's controlplane
+* One Grafana Deployment in `garden` namespace and two Deployments per shoot namespace(one exposed to the end users and one for the operators). Grafana is the UI component used in the logging stack.
+
+### How to access the logs
+We need to authenticate first in front of the Grafana ingress (Can be found in the Garden Dashboard in `Monitoring and Logging` section)
```

Comment:
```suggestion
We need to authenticate first in front of the Grafana ingress (can be found in the Garden Dashboard in `Monitoring and Logging` section)
```

Don't talk about the Gardener Dashboard here. Only talk about where the `Secret` can be found in the `garden-<project>` namespace.